openai/cli.py

```diff
@@ -1120,17 +1120,23 @@ def help(args):
     )
     sub.add_argument(
         "--project",
-        default="GPT-3",
-        help="""Name of the project where you're sending runs. By default, it is "GPT-3".""",
+        default="OpenAI-Fine-Tune",
+        help="""Name of the Weights & Biases project where you're sending runs. By default, it is "OpenAI-Fine-Tune".""",
     )
     sub.add_argument(
         "--entity",
-        help="Username or team name where you're sending runs. By default, your default entity is used, which is usually your username.",
+        help="Weights & Biases username or team name where you're sending runs. By default, your default entity is used, which is usually your username.",
     )
     sub.add_argument(
         "--force",
         action="store_true",
         help="Forces logging and overwrite existing wandb run of the same fine-tune.",
     )
+    sub.add_argument(
```

Comment:
One question - will this break existing workflows for users who expect to be able to use WandB sync with the old version of the fine tuning API? That might be okay, we'll just need to note as much in the release notes. My understanding is, if an existing user of the old API (`/v1/fine-tunes`) upgraded their `openai-python` version and tried to run fine tuning, unless they provided `--legacy`, this would break?

Reply:
Yep that’s correct